opta-mern-stack/client/opta-frontend-server.yaml

```diff
@@ -0,0 +1,16 @@
+name: mernfrontend
+environments:
+  - name: awsenv
+    path: "../awsenv.yaml"
```

Comment:
for GCP, this will need to be changed by the user, we should document this.
or alternatively, you can use the `--env ` flag to set it at runtime similar to this [example](https://github.com/run-x/opta-examples/tree/main/hello-app) -> if you do this re-test, as opta might do some validation in the env file.

Reply:
Ha, just realised our MongoDB module only supports AWS and Local, so removed all Google references from the example.

Reply:
Removed via hotfix in opta main repo as well.